Formula/s/stellar-core.rb

```diff
@@ -7,6 +7,11 @@ class StellarCore < Formula
   license "Apache-2.0"
   head "https://github.com/stellar/stellar-core.git", branch: "master"
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
  # labeled as "pre-release" on GitHub before the version is released, so it's
  # necessary to use the `GithubLatest` strategy.
  livecheck do
```

The `stable` URL isn't a GitHub release asset, so we should explain why `GithubLatest` is necessary here.